draft-packages/avatar/docs/Avatar.stories.tsx

```diff
@@ -1,9 +1,17 @@
 import * as React from "react"
 
+import { withDesign } from "storybook-addon-designs"
 import { Avatar } from "@kaizen/draft-avatar"
+import { figmaEmbed } from "../../../storybook/helpers"
```

Comment:
:( Welp, this makes the storybook tech debt a lot more obvious. I'll make an issue to add Storybook to yarn workspace so we can simplify these imports.